src/defs.h

```diff
@@ -477,6 +483,7 @@ struct sockaddr_store_t {
         sa_family_t ss_family;
         sockaddr_in addr4;
         sockaddr_in6 addr6;
+        sockaddr_un addr_un;
```

Comment:
Do not forget to reduce the size of this structure to store only `MAX_UDS_NAME` bytes of UNIX socket address.

Reply:
Here they define it by themselves 

![define2](https://user-images.githubusercontent.com/44922030/173329834-0ea013af-afed-4022-94df-c53d1721ac23.JPG)

Reply:
I hoped UNIX_PATH_MAX was portable. OK, keep using MAX_UDS_NAME.